styles/typography.md

```diff
@@ -78,6 +78,15 @@ title: Typography style
     Body: Noto sans, 20px, plain text
    </li>
   </ul>
+    <h2>Main page title</h2>
+  <p>When the H1 is the main title of a page, it includes a red bar as part of the Canada.ca brand.</p>
+  <p>Specifications for the red bar (gc-thickline):</p>
```

Comment:
@jmealing 

Can we please change this line to: 

Specifications for the red bar (formerly known as gc-thickline):

Reply:
Done